scripts/orb_utils.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-VERSION_REGEX="[0-9]\.[0-9]\.[0-9]"
+VERSION_REGEX="[0-9]*\.[0-9]*\.[0-9]*"
```

Comment:
@zephraph this should fix the mismatched version
```
$ YML_PATH=$(get_orb_path remote-docker)
$ VERSION_COMMENT=$(head -n 1 "$YML_PATH")
$ echo "$VERSION_COMMENT" | grep -o "$VERSION_REGEX"
0.1.1
$ VERSION_REGEX="[0-9]*\.[0-9]*\.[0-9]*"
$ echo "$VERSION_COMMENT" | grep -o "$VERSION_REGEX"
0.1.10
```

Reply:
Thanks for that!